app/Token.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use LaravelFCM\Message\OptionsBuilder;
+use LaravelFCM\Message\PayloadDataBuilder;
+use LaravelFCM\Message\PayloadNotificationBuilder;
+use FCM;
+use Carbon\Carbon;
+
+class Token extends Model
```

Comment:
FCM\DeviceToken

Reply:
With this use it does not work, there must be only FCM